.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "requests"]
 	path = requests
-	url = https://github.com/psf/requests.git
+	url = https://github.com/sailfishos-mirror/requests.git
```

Comment:
So Python module mirrors didn't end up being prefixed with python3?

Reply:
JFTR, yes, that was the decision.